modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -137,7 +137,17 @@ export class Utils implements BaseUtils {
    * @returns {EncodedTx} The decoded transaction.
    */
   decodeAlgoTxn(txnBytes: Uint8Array | string): EncodedTx {
-    const buffer = typeof txnBytes === 'string' ? Buffer.from(txnBytes, 'hex') : txnBytes;
+    let buffer;
+    if (typeof txnBytes === 'string') {
+      if (allHexChars(txnBytes)) {
+        buffer = Buffer.from(txnBytes, 'hex');
```

Comment:
In which cases tx should be hex from platform?

Reply:
From platform, it is always 'base64'. 
However, this method can support both 'hex' and 'base64'. The current tests in BitGoJS use 'hex' format and without support for both - BitGoJS unit tests fail.
We can update the data in resources to use base64 instead of hex and that can be a separate task. In either case, it is good to have this method support both formats